src/main/java/com/mars/review/ui/ReviewController.java

```diff
@@ -0,0 +1,36 @@
+package com.mars.review.ui;
+
+import com.mars.review.application.ReviewService;
+import com.mars.review.dto.ReviewRequest;
+import com.mars.review.dto.ReviewResponse;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@RestController
+@RequestMapping(value = "/api/review")
+public class ReviewController {
+    private final ReviewService reviewService;
+
+    @Autowired
+    public ReviewController(ReviewService reviewService) {
+        this.reviewService = reviewService;
+    }
+
+    @PostMapping()
+    public Long save(@RequestBody ReviewRequest req) {
+        return this.reviewService.save(req);
+    }
+
+    @GetMapping(value = "/{id}")
+    public List<ReviewResponse> findByMemberId(@PathVariable(name = "id") Long id){
+        return this.reviewService.findByMemberId(id).stream().map(review -> new ReviewResponse(review)).collect(Collectors.toList());
+    }
```

Comment:
이렇게 개행시키는 건 어떨까요? 가독성을 조금 챙겨볼 수 있을 것 같아요

```suggestion
    @GetMapping(value = "/{id}")
    public List<ReviewResponse> findByMemberId(@PathVariable(name = "id") Long id){
        return this.reviewService.findByMemberId(id).stream()
            .map(review -> new ReviewResponse(review))
            .collect(Collectors.toList());
    }
```

Reply:
넵 감사합니다!